src/Data/Biapplicative/Backwards.hs

```diff
@@ -0,0 +1,73 @@
+{-# LANGUAGE CPP #-}
```

Comment:
CPP isn't used in this module, so this language extension should be unnecessary.